tests/sles4sap/robot_fw.pm

```diff
@@ -67,6 +68,15 @@ sub run {
         select_console 'root-console';
     }
 
+    # It can only happen on sle product
+    # Only use by test not fully migrated to YAML
+    if (get_var('DISABLE_KDUMP')) {
+        record_info('Disabling kdump', 'Disabling kdump and crashkernel option');
```

Comment:
I can add this record info message to the function but the other functions don't have one.

Reply:
I think it is fine where you added it.